ReactWindows/ChakraBridge/ChakraHost.cpp

```diff
@@ -1,8 +1,13 @@
-﻿#include "pch.h"
+﻿#include <stdint.h>
+#include "pch.h"
```

Comment:
Pedantic question: is there a prefered style for sorting <> includes "" includes?